contributing.md

```diff
@@ -175,16 +175,19 @@ license your work under the [MIT License](http://en.wikipedia.org/wiki/MIT_Licen
 4. Don't manually update the version number in `package.json`. This is done using a Grunt task on deployment.
 
 <a name="grunt"></a>
-#### Using CoffeeScript and Grunt
+#### Grunt tasks: Running Tests and using CoffeeScript
 
 
 To install all development dependencies, in the project's root directory, run
 
     npm install
 
-Once you're configured, building the JavaScript from the command line is easy:
+Once you're configured, `grunt` tasks are available:
+
+    grunt test                 # translate CoffeeScript and run the tests in spec/
```

Comment:
Not sure about "translate". We speak of "build" later on, but maybe we should all replace that for "compile". 

> CoffeeScript is a little language that **compiles** into JavaScript.
> http://coffeescript.org